dfdatetime/java_time.py

```diff
@@ -3,14 +3,10 @@
 
 from __future__ import unicode_literals
 
-import decimal
-
-from dfdatetime import definitions
-from dfdatetime import interface
 from dfdatetime import posix_time
 
 
-class JavaTime(interface.DateTimeValues):
+class JavaTime(posix_time.PosixTimeInMilliseconds):
   """Java java.util.Date timestamp.
 
   The Java java.util.Date timestamp is a signed integer that contains the
```

Comment:
Please move the definitional wording here to the posixtimeinmilliseconds class.

Reply:
I don't agree - see the docstring for the PosixTime class for example. However, I've noticed that all the not-really-a-posix-timestamp-but-using-the-same-epoch classes have similar docstrings, so I'll adjust those in a separate PR